packages/client/src/app/challenge/components/ChallengeProgress.vue

```diff
@@ -11,7 +11,9 @@
           :dayNumber="i + 1"
           :todayNumber="todayNumber"
           :imageName="this.user.profile_picture"
-          :imagePath="`url('/assets/profile_pictures/${this.user.profile_picture}.png')`"
+          :imagePath="`url(${inject('serverBaseUrl')}'/assets/profile_pictures/${
```

Comment:
I think here's an error (some url elements are double), eventually it will be something like          :imagePath="`url(${inject('serverBaseUrl')}${this.user.profile_image_path})`"
but I can take it over after merging